plugins/modules/mysql_user.py

```diff
@@ -461,6 +477,14 @@ def main():
     if mysql_driver is None:
         module.fail_json(msg=mysql_driver_fail_msg)
 
+    if password_expire == "interval" and not password_expire_interval:
+        module.fail_json(msg="password_expire value interval \
+                         should be used with password_expire_interval")
+
+    if password_expire_interval and password_expire_interval < 1:
+        module.fail_json(msg="password_expire_interval value \
+                             should be positive number")
```

Comment:
```suggestion
    if password_expire == "interval" and not password_expire_interval:
        module.fail_json(msg="password_expire value interval \
                         should be used with password_expire_interval")

    if password_expire_interval and password_expire_interval < 1:
        module.fail_json(msg="password_expire_interval value \
                             should be positive number")
```
Can we use one of keys in AnsibleModue initialization instead? like `required_if` / `required_by`, etc.? See the keys at the bottom of the [doc section](https://docs.ansible.com/ansible/latest/dev_guide/developing_program_flow_modules.html#argument-spec).
Could you please check? if not possible, it's OK as is.

Reply:
can only move if statement of `password_expire`: `interval` is used together with `password_expire_interval`. But can't check `password_expire_interval` to be positive integer in `argument_spec`.

Reply:
@tompal3 

>can only move if statement of password_expire: interval is used together with password_expire_interval.

It'd be fine, could you please do it for this one?